tools/fds/server.cpp

```diff
@@ -569,6 +572,7 @@ int fastdds_discovery_server(
     }
 
     fastdds::rtps::GuidPrefix_t guid_prefix = participantQos.wire_protocol().prefix;
+    participantQos.transport().use_builtin_transports = true ? udp_server_initialized : false;
```

Comment:
This is the same as `participantQos.transport().use_builtin_transports = udp_server_initialized;`. Please check it is really what you wanted.